src/main/java/com/sforce/cd/apexUnit/client/testEngine/TestExecutor.java

```diff
@@ -25,8 +28,9 @@
 
 public class TestExecutor {
 	private static Logger LOG = LoggerFactory.getLogger(TestExecutor.class);
+	private static final int BATCH_SIZE = 200;
 
-	public ApexReportBean[] testExecutionFlow() {
+	/*public ApexReportBean[] testExecutionFlow() {
```

Comment:
@rduvvarapu would you remove the commented testExecutionFlow method since it's implemented below?